globus_cli/services/transfer.py

```diff
@@ -316,7 +316,7 @@ def check_completed():
 
     # Tasks start out sleepy
     if meow:
-        safeprint("""\
+        safeprint(r"""
```

Comment:
Is the extra space above the cat a breaking change? (It looks better, though, FWIW.)